### PR TITLE
DEVX-1730: Streams app development tutorial, mircroservices app fails…

### PR DESCRIPTION
**Pull request by ybyzek**
… to start up

### CODE REVIEW

File: microservices-orders/docker-compose.yml [hunk 1]
```diff
@@ -169,6 +169,7 @@ services:
       - "18894:18894"
     volumes:
       - $PWD/services-start.sh:/tmp/services-start.sh
+      - $PWD/config.env:/tmp/config.env
```
Comment (rspurgeon) [inline microservices-orders/docker-compose.yml]: There is no `config.env` file in $PWD, maybe you meant $PWD/../utils/config.env or similar?  The result of this testing the current way is that /tmp/config.env is an empty directory

```
streams@microservices-app:/build$ ls -la /tmp
total 16
drwxrwxrwt 1 root    root    4096 Apr 21 13:03 .
drwxr-xr-x 1 root    root    4096 Apr 21 13:03 ..
drwxr-xr-x 2 streams streams   64 Apr 21 13:03 config.env
drwxr-xr-x 2 streams streams 4096 Apr 21 13:04 hsperfdata_streams
-rwxr-xr-x 1 streams streams  893 Apr 21 12:52 services-start.sh

```

Comment (ybyzek) [reply microservices-orders/docker-compose.yml]: @rspurgeon the script (see below) sources `/tmp/config.env` so `$PWD` doesn't come into the picture (on the container itself)

Comment (rspurgeon) [reply microservices-orders/docker-compose.yml]: $PWD is the volume map on the host, but there is no config.env file on the host there, so the result is inside the container there is no config.env file at /tmp.  Docker makes an assumption that config.env is a directory if there is no file on the host at the volume point.  So the result inside the container is a directory, not a file, at /tmp/config.env.  As a result, the source command inside services-start.sh can't be sourcing anything meaningful.  Here is another command showing the issue from inside the container. 

```
streams@microservices-app:/build$ cat /tmp/config.env
cat: /tmp/config.env: Is a directory
```

Comment (ybyzek) [reply microservices-orders/docker-compose.yml]: @rspurgeon thanks for surfacing this.  Source of confusion was an unchecked file on my local host.  Can you please verify PR with latest commit?

Comment (rspurgeon) [reply microservices-orders/docker-compose.yml]: 👍 
```
docker-compose exec microservices-app bash
streams@microservices-app:/build$ cat /tmp/config.env
#####################################################
# Values in this file will be used to replace variables in script files
...
```

**Review by rspurgeon — APPROVED** [review]
LGTM

**Pull request merged**